resources/core/templates/istio-injection-label.yaml

```diff
@@ -20,8 +20,9 @@ spec:
         command:
         - kubectl
         - label
+        - --overwrite
```

Comment:
Required when reinstalling `core` chart, this label already exists and kubectl fails without --overwrite flag